adbe/main.py

```diff
@@ -270,7 +270,10 @@ def main():
 
     # apps list
     elif args['apps'] and args['list'] and args['all']:
-        adb_enhanced.list_all_apps()
+        all_packages, err_msg, err = adb_enhanced.get_list_all_apps()
```

Comment:
I like `adb_enhanced.list_all_apps()` better here. main.py is already too large, let's not put more logic in it. It should only call functions in adb_enhanced.